pkg/proc/gdbserial/gdbserver.go

```diff
@@ -1042,7 +1042,7 @@ func (p *gdbProcess) getCtrlC(cctx *proc.ContinueOnceContext) bool {
 
 // Detach will detach from the target process,
 // if 'kill' is true it will also kill the process.
-func (p *gdbProcess) Detach(kill bool) error {
+func (p *gdbProcess) Detach(pid int, kill bool) error {
```

Comment:
Can you add a comment to the doc string about why `pid` is unused (because process groups aren't implemented with this backend), and maybe prefix with `_` to make that clear in the body.

Reply:
Done